api/user/schema.py

```diff
@@ -212,11 +212,11 @@ def mutate(self, info, **kwargs):
         location_id = kwargs['location_id']
         query_user = User.get_query(info)
         user = query_user.filter(
-          UserModel.state == "active", UserModel.email == email).first()
+            UserModel.state == "active", UserModel.email == email).first()
```

Comment:
You introduced new changes which do not have any effect. This is also on line 219. Please check on this.

Reply:
Flake8 suggested those changes.

Reply:
Okay then.

Reply:
@cop1fab Kindly invalidate this change. It is not part of the task